Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM node:18 AS vendors
+
+COPY . /srv/umap
+
+WORKDIR /srv/umap
+
+RUN make installjs
+
+RUN make vendors
+
+FROM python:3.8-slim
+
+ENV PYTHONUNBUFFERED=1 \
+    UMAP_SETTINGS=/srv/umap/umap/settings/docker.py \
+    PORT=8000
+
+RUN mkdir -p /srv/umap/data && \
+    mkdir -p /srv/umap/uploads
+
+COPY . /srv/umap
+
+COPY --from=vendors /srv/umap/umap/static/umap/vendors /srv/umap/umap/static/umap/vendors
```

Comment:
This should not be needed if using a stable release instead of the sources

Reply:
Maybe @pommi can share some of his work as he followed the steps in the installation.